enemy-test/src/enemy_monster.c

```diff
@@ -24,71 +24,58 @@ void display_enemy(ENEMY* unit, UINT8 xpos, UINT8 ypos) {
 	
 	// Initialize left sprite
 	set_sprite_tile(unit->sprite_id, unit->enemy_sprite_l);
-	move_sprite(unit->sprite_id, xpos, ypos);
+	move_sprite(unit->sprite_id, xpos - 8, ypos - 16);
 	
 	// Initialize right sprite
 	set_sprite_tile(unit->sprite_id + 1, unit->enemy_sprite_r);
-	move_sprite(unit->sprite_id + 1, xpos + 8, ypos);
+	move_sprite(unit->sprite_id + 1, xpos, ypos - 16);

```

Comment:
I probably should have been clearer, but you actually need to _add_ your offset (the gameboy expects the origins of sprites to be at the bottom-right corner)

Reply:
Yup, I realized shortly after committing. Fixed.